tap-gui/configurator/external-plugins.hbs.md

```diff
@@ -8,7 +8,682 @@ This topic tells you how to add external plug-ins to Tanzu Developer Portal
 As mentioned in [Build your Customized Tanzu Developer Portal with Configurator](building.hbs.md#prereqs),
 to use any type of external plug-in, ensure that it is in an npm registry. This registry can be your
 own private registry or a plug-in registry if you intend to use a third-party or community plug-in.
```

Comment:
@mstergianis QQ: are we enabling private registries in this release - I thought we only do public npmjs.org in 1.6.1

Reply:
Hmm, with these levels of workarounds a customer actually could add their own private registry by modifying the verdaccio config (without publishing directly to verdaccio) and uploading a custom builder... But yeah you're right maybe I shouldn't mention all of that.

Though FWIW I'm [reasonably confident](https://vmware.slack.com/archives/C03QP5Z3VJ4/p1689963028275329) that npmjs connectivity doesn't even work

Reply:
I was able to build with a custom plugin that I deployed to npmjs.

Reply:
Interesting, I was seeing issues with the verdaccio cache lifetime of `2y`. But I don't understand verdaccio all that well yet, maybe I was pointing the blame poorly. Or maybe if verdaccio is already aware of that package (metadata and maybe some cached version(s)) then it doesn't bother to download metadata. But if verdaccio has no notion of that package it will download metadata from upstream?

Anyways, more to learn for me

Reply:
It worked for me as well from npmjs so it definitely should work for public npmjs published wrappers